slices/libtinfo6.yaml

```diff
@@ -0,0 +1,8 @@
+package: libtinfo6
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+    contents:
+      /lib/*-linux-*/libtinfo.so.6*:
```

Comment:
Perhaps we should add `/usr/lib/*-linux-*/libtic.so.6*` as done in https://github.com/canonical/chisel-releases/blob/ubuntu-20.04/slices/libtinfo6.yaml.

Reply:
Yes, indeed, added, thanks!